go/cmd/vttlstest/vttlstest.go

```diff
@@ -60,49 +58,58 @@ func init() {
 		"CreateSignedCert":     cmdCreateSignedCert,
 		"RevokeCert":           cmdRevokeCert,
 	}
-}
 
-var (
-	root = flag.String("root", ".", "root directory for certificates and keys")
-)
+	servenv.OnParseFor("vttlstest", func(fs *flag.FlagSet) {
```

Comment:
this can just be `OnParse` (this is a `main` package, so nothing can import it, so there is no difference between "scoped" and "global" registration here)

Reply:
Makes sense, will do. Thanks!